docs/source/benchmarking/materials_science_and_engineering/oiematerials.rst

```diff
@@ -1,12 +1,12 @@
 Open Innovation Environment (OIE) domain ontologies, Materials module (OIEMaterials)
```

Comment:
convert `Open Innovation Environment (OIE) domain ontologies, Materials module (OIEMaterials)` to `Open Innovation Environment Materials (OIEMaterials)`